cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/sinks/NGSIPostgreSQLSink.java

```diff
@@ -547,7 +547,7 @@ public String buildSchemaName(String service, String subService) throws CygnusBa
                 case DMBYENTITYDATABASESCHEMA:
```

Comment:
An entry in CHANGES_NEXT_RELEASE file should be included. I'd suggest:

```
- [cygnus-ngsi] Removes "_ " in schema name for DM -schema family (#2201)
```

Reply:
Added in 59c4a86